javalib/src/main/scala/java/time/DateTimeException.scala

```diff
@@ -0,0 +1,4 @@
+package java.time
+
+class DateTimeException(message: String, cause: Throwable = null)
```

Comment:
Parameters cannot have default values in JDK classes. You need to declare separate overloads. (default params and overloads are not binary compatible)